active/progress/progress.md

```diff
@@ -0,0 +1,111 @@
+<!--See comments in Markdown for how to use this spec template-> 
+
+<!-- The purpose of this spec is to describe a new feature and
+its APIs that make up a new feature in WinUI. -->
+
+<!-- There are two audiences for the spec. The first are people
+that want to evaluate and give feedback on the API, as part of
+the submission process.  When it's complete
+it will be incorporated into the public documentation at
+docs.microsoft.com (http://docs.microsoft.com/uwp/toolkits/winui/).
+Hopefully we'll be able to copy it mostly verbatim.
+So the second audience is everyone that reads there to learn how
+and why to use this API. -->
+
+# Background
+<!-- Use this section to provide background context for the new API(s) 
+in this spec. -->
+
+<!-- This section and the appendix are the only sections that likely
+do not get copied to docs.microsoft.com; they're just an aid to reading this spec. -->
+
+<!-- If you're modifying an existing API, included a link here to the
+existing page(s) -->
+
+<!-- For example, this section is a place to explain why you're adding this API rather than
+modifying an existing API. -->
+
+<!-- For example, this is a place to provide a brief explanation of some dependent
+area, just explanation enough to understand this new API, rather than telling
+the reader "go read 100 pages of background information posted at ...". -->
+
+>See proposal documents for:
+>
+> [determinate mode for ProgressRing](https://github.com/microsoft/microsoft-ui-xaml/issues/688) 
+>
+> [ProgressRing style](https://github.com/microsoft/microsoft-ui-xaml/issues/837)
+>
+> [Guidance for Progress controls](https://github.com/microsoft/microsoft-ui-xaml/issues/880)
+
+Progress controls indicate to a user that an operation is occuring. This includes ProgressBar and ProgressRing controls. ProgressBar has both a determinate and indeterminate state. ProgressRing only has an indeterminate state.
```

Comment:
Once ProgressRing has this new property there would be closer functional parity between ProgressRing and ProgressBar. What guidance should we give to developers regarding when to use ProgressRing vs. ProgressBar?  Would it strictly be a stylistic difference now?  

If so, would we ideally just have a single control instead with two different views (control templates)?

Reply:
If you combine ProgressRing and ProgressBar into a single Progress Control - you could have an enum to switch between a Ring or Bar layout.

You would also have to factor in how values like "StartingAngle" and Thickness would be used by the Bar layout.

Reply:
It might make sense to eventually do this (combine into one control); however, we also have a goal of making the adoption of WinUI 3.0 (especially when coming from UWP XAML that ships as a DLL inside Windows) as seamless as possible.  That means minimal API changes/churn, so I wonder if combination into a single control would be a better target for WinUI 4, rather than an earlier release.  Something to think about, anyway.

Reply:
There is a new ScrollViewer control that currently coexists with the native UWP one.  Perhaps a combined control could come alongside the existing two before replacing it eventually with intellisense suggesting setting the IsIndeterminate property to true?

Reply:
Yeah, that's definitely an option we could consider too.  Not sure if I like that better than just having them separate, and then changing to combined in one fell swoop.  But worth some debate/thought at least.

Reply:
Our team has discussed this and for now we'll keep the controls separate